pkg/binfmt/Makefile

```diff
@@ -1,15 +1,4 @@
-.PHONY: tag push
-default: push
+include ../package.mk
 
-ORG?=linuxkit
 IMAGE=binfmt
-DEPS=Dockerfile Makefile main.go $(wildcard etc/binmft.d/*)
-
-HASH?=$(shell git ls-tree HEAD -- ../$(notdir $(CURDIR)) | awk '{print $$3}')
-
-tag: $(DEPS)
-	docker build --no-cache --network=none -t $(ORG)/$(IMAGE):$(HASH) .
-
-push: tag
-	DOCKER_CONTENT_TRUST=1 docker pull $(ORG)/$(IMAGE):$(HASH) || \
-	DOCKER_CONTENT_TRUST=1 docker push $(ORG)/$(IMAGE):$(HASH)
+DEPS=$(DEPS) main.go $(wildcard etc/binmft.d/*)
```

Comment:
You can write `DEPS += main.go ...` here.

Might it be better to have `package.mk` define `BASE_DEPS = Makefile Dockerfile` and then have `tag: $(BASE_DEPS) $(DEPS)`, that way in the default case noone needs to worry about accidentally overwriting the base deps (but you can still zero them if you want). Or perhaps this will be more complicated/confusing for users by introducing a two tier set of deps.